docs/Windows.md

```diff
@@ -1,43 +1,172 @@
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.apinotes" -o "%VCToolsInstallDir%\include\visualc.apinotes"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/winsdk.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\um\module.modulemap"
 ```
-3. You will need to add the ICU libraries for the target.  The nightlies are built against ICU 64.2 from the ICU project.  You can download the binaries for that from http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip.
+
+Close this instance of the Native Tools Command Prompt after those commands have been successfully executed.
+
+#### Swift Toolchain & Platform SDK concepts
+
+The installation instructions that follow will result in a directory tree that has a well thought-out structure. To understand this directory structure better you may consult  <https://github.com/compnerd/windows-swift/blob/master/docs/details.md>. But first, you might want to continue with the installation.
+
+#### Downloading the nightlies
+
+1. Go to  <https://dev.azure.com/compnerd/windows-swift>.
+2. Choose `Pipelines` > `Pipelines` from the left of the dashboard.
+3. Choose 'Runs'.
+4. Use the filter symbol to search for "VS2017".
+5. In the results list, scroll down until you see the first successful build (with a green OK symbol) and click on it.
+6. Under "Artifacts", click on the "... published" link.
+7. Download windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi by clicking on the appropriate down-arrows on the right. Be sure to really download these files from the same build (i.e. do not switch the build for the next download, and be careful when updating)! These files will be downloaded as zip files. Unless they are not automatically unzipped during the download process, unzip them to obtain *.msi files in the extracted directories.
+
+#### Installing the nightlies
+
+These *.msi files install the files to C:\Library. The complete Library directory can later be copied to a different location to be used there.
+
+Double-click windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi to run the according installations, one by one. (The installation process started by windows-toolchain-amd64.msi might take a while. Each of those installations can be deinstalled separately via the Windows "Apps & Features" settings. Such an deinstallation leaves all files in C:\Library that are not part of the according installation.)
+
+Those installations to not add any values to the PATH environment variable. The PATH environment variable has to adjusted separatly (see below).
+
+The directories created by those installations are as follows:
+
+- windows-toolchain-amd64.msi: `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain`
+- windows-sdk.msi: `C:\Library\Developer\Platforms\Windows.platform`
```

Comment:
I don't think that this path is correct nor needed.  It may make it easier if you added the final location of the import libraries to the `LIB` variable, but, generally, this is implicitly taken care of by the `-sdk` parameter.